File: pinyougou_interface/src/main/java/cn/itcast/core/service/OrderService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.order.Order;
import entity.PageResult;
import vo.OrderVo;

import java.util.Map;

public interface OrderService {
    void add(Order order);

    PageResult searchOrder(Integer pageNo, Integer rows, OrderVo orderVo);
}

File: pinyougou_web_user/src/main/webapp/js/service/orderService.js
// 订单业务层
app.service("orderService",function ($http) {
    // 分页查询订单
    this.searchOrder = function (pageNo,rows,orderVo) {
        return $http.post("../order/searchOrder.do?pageNo="+pageNo+"&rows="+rows,orderVo);
    }
})